benchmarks/oss.py

```diff
@@ -119,7 +119,7 @@ def closure():
     print(f"[{dist.get_rank()}] : Mean speed: {mean:.2f} +/- {std:.2f}")
 
     if use_oss and check_regression and dist.get_rank() == 0:
-        assert (mean - 3.0 * std) < reference_speed, "Speed regression detected"
+        assert (mean + 3.0 * std) > reference_speed, "Speed regression detected"
```

Comment:
this was wrong, fixed in another PR but might as well be fixed here (I was bumping into this locally)

Reply:
(we want the speed to increase, not decrease.. the test initially made sense when comparing runtime, now that it compares frames per second higher is better)